src/library.js

```diff
@@ -2899,6 +2899,20 @@ mergeInto(LibraryManager.library, {
       _free = prev_free;
     }
   },
+
+  _emscripten_sanitizer_use_colors: () => {
+    var setting = Module['printWithColors'];
+    if (setting != null) {
```

Comment:
This is just moved code, but it should probably be `!== undefined`. But it does work as expected thanks to using `!=` and not `!==` atm. Anyhow, unrelated to this PR.